ibmcloud/image/verify.sh

```diff
@@ -99,3 +103,4 @@ os.write(fd, b"\035\n") # send ^[
 os.write(1, b"\n")
 sys.exit(exit_status)
 END
+fi
```

Comment:
Then, remove this line.

Reply:
Thank you so much for reviewing these changes! I've made some further changes based on your comments :)